java-manta-client-unshaded/src/main/java/com/joyent/manta/http/MantaConnectionFactory.java

```diff
@@ -379,14 +268,14 @@ protected HttpClientBuilder createBuilder(final HttpSignatureAuthScheme authSche
         final HttpClientBuilder builder = HttpClients.custom()
                 .disableAuthCaching()
                 .disableCookieManagement()
+                .setDefaultHeaders(HEADERS)
```

Comment:
Was this just re-ordered for sorting, or is there meaning to the move?

Reply:
This branch was extracted from a larger set of changes that included reorganizing the setters so that some would be skipped when an existing `HttpClientBuilder` would be passed in.